em-app/src/platform/nitro.rs

```diff
@@ -29,38 +70,38 @@ pub(crate) fn get_remote_attestation_parameters(
     let extensions = alt_names.and_then(|names| {
         Some(vec![(pkix::oid::subjectAltName.clone(), false, pkix::yasna::construct_der(|w| DnsAltNames { names }.write(w)).into())])
     });
-    
+
     let csr_pem = get_csr_common_name(signer, &common_name, attributes, &extensions)?;
 
     Ok((None, None, csr_pem))
 }
 
+fn pcr0_from_env() -> Result<Vec<u8>> {
+    let value = std::env::var("ENCLAVEOS_PCR0").map_err(|_| Error::AttestationCertInternal("Missing 'ENCLAVEOS_PCR0' inside debug enclave.".into()))?;
```

Comment:
An environment variable was chosen here because debug enclaves aren’t secure in the first place (RAM can be decrypted). By putting PCR0 we have an identity that can be used to determine policy in insecure accounts that allow debug enclaves.

I see your point about generating build-time ids instead, which we could pursue, but that would make deterministic builds a bit harder for ease of use and I’m not sure we want to make that tradeoff for people.

Reply:
Also, by only including it in a separate extension, clients (including us) can choose to never trust enclaves including this separate PCR0 extension.